java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -384,6 +385,12 @@ public CompletableFuture<String[]> lrange(@NonNull String key, long start, long
                 response -> castArray(handleArrayOrNullResponse(response), String.class));
     }
 
+    @Override
+    public CompletableFuture<String> lindex(@NonNull String key, int index) {
```

Comment:
int or long? How big redis indices are? Please check and update if needed
```suggestion
    public CompletableFuture<String> lindex(@NonNull String key, long index) {
```

Reply:
Fixing it.